variables.tf

```diff
@@ -34,6 +28,15 @@ variable "aws_okta_group_ids" {
   description = "List of Okta group IDs that should be assigned the AWS SSO Okta app"
 }
 
+variable "aws_regions" {
```

Comment:
This variable name does not really represent the description, maybe aws_region_restrictions ?

Reply:
Makes sense...I'll update it